crowdsec-docs/unversioned/getting_started/post_installation/acquisition.mdx

```diff
@@ -0,0 +1,252 @@
+---
+id: acquisition
+title: Acquisition
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+import CodeBlock from '@theme/CodeBlock';
+
+# Acquisition
+
+By default when CrowdSec is installed it will attempt to detect the running services and acquire the appropriate parsers and scenarios.
+
+However, we should check that this detection worked or you may want to manually acquire parsers and scenarios for services that are not detected.
+
+## What services are already detected?
+
+To find what services are already detected, you can use the `cscli` command line tool.
+
+```bash
+cscli collections list
+```
+
+:::info
+Collections are packages of parsers and scenarios that are used to detect security threats.
+:::
+
+### How to interpret the output
+
+The output will show a list of collections that are already installed. For example `crowdsecurity/sshd` is the collection for the `sshd` service.
+
+## Are the detected services working correctly?
+
+To check if the detected services are working correctly, you can use the `cscli` command line tool.
+
+```bash
+cscli collections inspect <collection_name>
```

Comment:
what is tricky here is that not all collections are link to datasource. For example `base-http` is part of nginx and doesn't have a datasource associated. I'm afraid it might confuse users. Do you think trying to explain the relevant section of `cscli metrics` might be easier to understand ?

Reply:
After discussion, we assume this won't be merged before 1.6.1 and we can then rely on the fancy new `cscli metrics show X` command